### PR TITLE
Improve i18n flow

### PR DESCRIPTION
**Pull request by zekth**
This improves the i18n flow by adding a script to check the completion of the translations. Once all the translations will be updated and fixed we can hook github ci to create an issue once there is new entries in the master translations.

Feel free to comment.

### CODE REVIEW

**Comment by ranjib** [comment]
👍 nice. love it

**Comment by zekth** [comment]
> 👍 nice. love it

@ranjib is the report in the CI good for you? There is a detail and a resume at the end. For me it's enough ATM.

Also i can refactor the Travis to move to github CI to have everything centralized and i can also optimize the builds.

**Comment by ranjib** [comment]
I like it. Fix the build and merge. I'll onboard go plumbing. Why we have to refactor travis ? I'll prefer to keep it untouched till we have github ci running smoothly for sometime

**Comment by zekth** [comment]
> Fix the build and merge

So the Translations CI is exiting with 1 if there is missing translations ATM. Do we want to exit 1 in case of fatal error instead?

File: build/translations/csv-sync.js [hunk 1]
```diff
@@ -1,37 +1,105 @@
...
+
+function deleteDeprecatedTranslations(slaveArray, masterArray) {
+  const masterKeys = masterArray.map(x => x.Key)
+  slaveArray = slaveArray.filter(x => masterKeys.includes(x.Key))
+  return slaveArray
+}
+
+function syncTranslationFiles(masterTranslations) {
+  files.forEach(f => {
+    let slave = fs.readFileSync(`${PATH_CSV}${f}`, 'utf8')
```
Comment (ranjib) [inline build/translations/csv-sync.js]: dialect, not slave

**Comment by ranjib** [comment]
Just add the i18n strings in the translation files, that should pass the build. We can have separate target to check if the individual dialects have their respective values populated or not. That does not need to be verified at CI level. It will be nice to sort keys of these file and rewrite if needed.

**Comment by zekth** [comment]
> Just add the i18n strings in the translation files, that should pass the build. We can have separate target to check if the individual dialects have their respective values populated or not. That does not need to be verified at CI level. It will be nice to sort keys of these file and rewrite if needed.

I think there is a missunderstood in here. The build process is still the same and is not armed by this PR. This CI job is only checking the completion of the translations, it does not have any impact on the build itself.

File: build/translations/csv-sync.js (hunk 1 above)
Comment (zekth) [reply build/translations/csv-sync.js]: so by this, i guess `master` should be renamed as `reference` ?

**Comment by ranjib** [comment]
@zekth i was talking about consolidating all CI works under github CI from travis,

**Comment by zekth** [comment]
So now the CI will only break in case of missing key in Dialect from the reference language which is english. When you do the translations:sync now it will synchronize but also reorder the keys in the CSV to make everything more maintainable.

@ranjib i think it fits our needs

**Comment by zekth** [comment]
Merging it @ranjib ? i don't get why the build is red. Is the ci flaky?

**Comment by ranjib** [comment]
There’s a test with race condition :-/. Let me fix it. It’s annoying me for
some time, and it’s introduced by me :-(

**Pull request merged**